packages/cli/src/commands/deploy.ts

```diff
@@ -21,16 +55,14 @@ export const generateBundleZip = async (rootDir: string) => {
   try {
     await access(distDir);
   } catch {
-    consola.error(`Dist directory not found: ${distDir}`);
-    process.exit(1);
+    throw new Error(`Dist directory not found: ${distDir}`);
```

Comment:
Nice, thanks for switching to this! I like this pattern where we `throw` errors in individual functions (which makes them easier to test as units), let the errors bubble up to the action callback, and then display them with `consola.error` in the action callback's `catch`.